yt_dlp/extractor/yandexvideo.py

```diff
@@ -268,35 +270,75 @@ class ZenYandexChannelIE(InfoExtractor):
         'url': 'https://zen.yandex.ru/tok_media',
         'info_dict': {
             'id': 'tok_media',
+            'description': 'md5:84d2fb177e478e44690f131805d7c842',
+            'title': 'ТОК',
         },
         'playlist_mincount': 169,
     }, {
         'url': 'https://zen.yandex.ru/id/606fd806cc13cb3c58c05cf5',
         'info_dict': {
             'id': '606fd806cc13cb3c58c05cf5',
+            'description': 'md5:517b7c97d8ca92e940f5af65448fd928',
+            'title': 'AcademeG DailyStream',
         },
         'playlist_mincount': 657,
     }]
 
-    def _entries(self, id, url):
-        webpage = self._download_webpage(url, id)
-        data_json = self._parse_json(re.findall(r'var\s?data\s?=\s?({.+?})\s?;', webpage)[-1], id)
-        for key in data_json.keys():
-            if key.startswith('__serverState__'):
-                data_json = data_json[key]
-        items = list(try_get(data_json, lambda x: x['feed']['items'], dict).values())
-        more = try_get(data_json, lambda x: x['links']['more']) or None
+    def _entries(self, channel_title, server_state_json, server_settings_json):
+        items = list(try_get(server_state_json, lambda x: x['feed']['items'], dict).values())
```

Comment:
try_get will return None if it's not found or not dict.
In that case doing doing None.values() will throw runtime error